app/README.md

```diff
@@ -37,3 +42,60 @@ npm run start:app
+
+Within these tabs, the left panel navigation allows for selecting artifact types or searching (Repository only) for artifacts so that they can be browsed for viewing or actions.
+
+<img src="./static/navigation.png" alt="Screenshot of navigation" width="250"/>
+
+## Landing Page
+The landing page can be accessed on first load or by clicking the Measure Repository home button on the top left. This page links to resources to learn more about the Measure Repository and shows existing FHIR service capabilities.
+
+## Repository Tab
+The Repository tab displays sets of artifacts from the FHIR Measure Repository Service. Options allow for viewing resource details or reviewing the resource.
+
+<img src="./static/repository-options.png" alt="Screenshot of repository options" width="75"/>
+
+### Viewing
+-	Measure: Resource details include options to see the JSON, Data Requirements, or Dependencies.        
+-	Library: Resource details include options to see the JSON or Narrative.  
+-	Drafting: Both resource views allow the user to create a draft from the artifact. This action copies the existing artifact and any children it owns to the draft Authoring space with an incremented version to logically differentiate it from the original artifact.
+
+### Reviewing
+Reviewing a resource provides options for viewing existing comments on the resource or adding a new comment. (Note: this is currently partially implemented and requires updates for viewing comments.)
+
+## Authoring Tab
+The Authoring tab gives options for creating a new draft artifact from scratch or starting from a copy of an existing artifact from the FHIR Measure Repository. 
+
+<img src="./static/draft-artifact.png" alt="Screenshot of create draft artifact" width="350"/>
+
+The left navigation resource type selection displays sets of artifacts that are in progress, being drafted for potential addition to the FHIR Measure Repository. Options allow for editing, reviewing, or deleting a draft resource.
+
+<img src="./static/authoring-options.png" alt="Screenshot of authoring options" width="120"/>
+
+### Editing
+-	Updating: A Measure or Library allows changes to basic resource fields:
+    - url
+    - identifier value
+    - identifier system
+    - name
+    - title
+    - description
+    - library (Measure only)
+- Releasing: Both resource types may be released from the editing page. This action publishes the draft artifact and any children it owns to the Repository space, adding them to the FHIR Measure Repository and giving them an active status. The draft artifact(s) will be removed from the Authoring space.      
```

Comment:
Do we want this to be its own section?

Reply:
I've kind of organized everything along the current navigation hierarchy. Releasing is a sub-function only available once you start editing, so that's why it's in the editing section. Could potentially reorganize if there's a different organizational scheme that makes more sense!

Reply:
```suggestion
- Releasing: Both resource types may be released from the editing page. This action publishes the draft artifact and any children it owns to the Repository space, adding them to the FHIR Measure Repository and giving them an active status. The draft artifact(s) will be removed from the Authoring space. See the [release operation specification](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#release).
```